### PR TITLE
Update and run borrow-schema.py

### LINKED ISSUE #139: Update to OCDS 1.1.4
After updating OCDS for PPPs to OCDS 1.1.4, we can run `borrow-schema.py` to update this to OCDS 1.1.4 (just changes code/field descriptions).

### PR DESCRIPTION
**Pull request by jpmckinney**
closes #139, replaces #167

documentType.csv

* needsAssessment and contractSchedule are re-ordered but otherwise unchanged
* plannedProcurementNotice is added from OCDS
* Updates descriptions from OCDS

partyRole.csv

* interestedParty is added from OCDS
* Updates descriptions from OCDS

project-schema.json

* Location.id is preserved and required

### CODE REVIEW

**Comment by jpmckinney** [comment]
To review, you might prefer to click on individual commits, since this includes #173.

**Review by duncandewhurst — APPROVED** [review]
Thanks for updating the script.

Changes to script, codelists and schema look good.

Let's wait until we've merged the updated typos PR before merging this.

**Comment by jpmckinney** [comment]
I've removed the commits from the typos PR, so I'll merge this as approved once tests are done.

**Pull request merged**
